documentation/polymer-templates/tutorial-styling-polymer-templates.asciidoc

```diff
@@ -0,0 +1,44 @@
+---
+title: PolymerTemplate, Styling Templates
+order: 12
+layout: page
+---
+
+= PolymerTemplate, Styling Templates
+
+*Usage of polymer based templates is deprecated since Vaadin 18 and we recommend using Lit based templates instead.*
```

Comment:
- “Polymer” should be capitalised
- Better break onto two separate sentences
- I guess LitElement should be mentioned with a complete name, otherwise it is confusing, as there are also `lit-html` templates

```suggestion
*Usage of Polymer based templates is deprecated since Vaadin 18. It is recommended to use LitElement based templates instead.*
```